Header header
string child_frame_id
uint32 id
uint32 score
uint32 contrast
geometry_msgs/Pose pose
